src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/OffenderImageRepository.java

```diff
@@ -1,6 +1,5 @@
 package uk.gov.justice.hmpps.prison.repository.jpa.repository;
 
-
```

Comment:
Redundant

Reply:
Just removing additional blank line in passing.